pkg/alertmanager/distributor.go

```diff
@@ -75,7 +75,13 @@ func (d *Distributor) isQuorumWritePath(p string) bool {
 }
 
 func (d *Distributor) isUnaryWritePath(p string) bool {
-	return strings.HasSuffix(p, "/silences")
+	if strings.HasSuffix(p, "/silences") {
+		return true
+	}
+	if strings.HasSuffix(p, "/api/v1/grafana/receivers/test") {
+		return true
+	}
+	return false
```

Comment:
Nit, maybe we could collapse them into a single `if`
```suggestion
	return strings.HasSuffix(p, "/silences") || strings.HasSuffix(p, "/silence")
```